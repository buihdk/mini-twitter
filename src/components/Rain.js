import React from 'react';
import '../styles/Rain.css';

const Rain = () => {
  return (
    <div className="rain-wrapper">
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
    </div>
  );
};

export default Rain;
