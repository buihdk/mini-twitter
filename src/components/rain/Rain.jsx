import React from 'react';

import './Rain.css';

const Rain = () => {
  return (
    <div className="rain-wrapper">
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
      <i className="rain" />
    </div>
  );
};

export default Rain;
