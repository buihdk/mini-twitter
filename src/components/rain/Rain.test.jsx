import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import Rain from './Rain';

describe('Rain', () => {
  test('renders without crashing', () => {
    const wrapper = shallow(<Rain />);

    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
