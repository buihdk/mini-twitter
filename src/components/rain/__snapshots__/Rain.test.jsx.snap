// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rain renders without crashing 1`] = `
<div
  className="rain-wrapper"
>
  <i
    className="rain"
    key="0"
  />
  <i
    className="rain"
    key="1"
  />
  <i
    className="rain"
    key="2"
  />
  <i
    className="rain"
    key="3"
  />
  <i
    className="rain"
    key="4"
  />
  <i
    className="rain"
    key="5"
  />
  <i
    className="rain"
    key="6"
  />
  <i
    className="rain"
    key="7"
  />
  <i
    className="rain"
    key="8"
  />
  <i
    className="rain"
    key="9"
  />
  <i
    className="rain"
    key="10"
  />
  <i
    className="rain"
    key="11"
  />
  <i
    className="rain"
    key="12"
  />
  <i
    className="rain"
    key="13"
  />
  <i
    className="rain"
    key="14"
  />
  <i
    className="rain"
    key="15"
  />
  <i
    className="rain"
    key="16"
  />
  <i
    className="rain"
    key="17"
  />
  <i
    className="rain"
    key="18"
  />
  <i
    className="rain"
    key="19"
  />
  <i
    className="rain"
    key="20"
  />
  <i
    className="rain"
    key="21"
  />
  <i
    className="rain"
    key="22"
  />
  <i
    className="rain"
    key="23"
  />
  <i
    className="rain"
    key="24"
  />
  <i
    className="rain"
    key="25"
  />
  <i
    className="rain"
    key="26"
  />
  <i
    className="rain"
    key="27"
  />
  <i
    className="rain"
    key="28"
  />
  <i
    className="rain"
    key="29"
  />
  <i
    className="rain"
    key="30"
  />
  <i
    className="rain"
    key="31"
  />
  <i
    className="rain"
    key="32"
  />
  <i
    className="rain"
    key="33"
  />
  <i
    className="rain"
    key="34"
  />
  <i
    className="rain"
    key="35"
  />
  <i
    className="rain"
    key="36"
  />
  <i
    className="rain"
    key="37"
  />
  <i
    className="rain"
    key="38"
  />
  <i
    className="rain"
    key="39"
  />
  <i
    className="rain"
    key="40"
  />
  <i
    className="rain"
    key="41"
  />
  <i
    className="rain"
    key="42"
  />
  <i
    className="rain"
    key="43"
  />
  <i
    className="rain"
    key="44"
  />
  <i
    className="rain"
    key="45"
  />
  <i
    className="rain"
    key="46"
  />
  <i
    className="rain"
    key="47"
  />
  <i
    className="rain"
    key="48"
  />
  <i
    className="rain"
    key="49"
  />
  <i
    className="rain"
    key="50"
  />
  <i
    className="rain"
    key="51"
  />
  <i
    className="rain"
    key="52"
  />
  <i
    className="rain"
    key="53"
  />
  <i
    className="rain"
    key="54"
  />
  <i
    className="rain"
    key="55"
  />
  <i
    className="rain"
    key="56"
  />
  <i
    className="rain"
    key="57"
  />
  <i
    className="rain"
    key="58"
  />
  <i
    className="rain"
    key="59"
  />
  <i
    className="rain"
    key="60"
  />
  <i
    className="rain"
    key="61"
  />
  <i
    className="rain"
    key="62"
  />
  <i
    className="rain"
    key="63"
  />
  <i
    className="rain"
    key="64"
  />
  <i
    className="rain"
    key="65"
  />
  <i
    className="rain"
    key="66"
  />
  <i
    className="rain"
    key="67"
  />
  <i
    className="rain"
    key="68"
  />
  <i
    className="rain"
    key="69"
  />
  <i
    className="rain"
    key="70"
  />
  <i
    className="rain"
    key="71"
  />
  <i
    className="rain"
    key="72"
  />
  <i
    className="rain"
    key="73"
  />
  <i
    className="rain"
    key="74"
  />
  <i
    className="rain"
    key="75"
  />
  <i
    className="rain"
    key="76"
  />
  <i
    className="rain"
    key="77"
  />
  <i
    className="rain"
    key="78"
  />
  <i
    className="rain"
    key="79"
  />
  <i
    className="rain"
    key="80"
  />
  <i
    className="rain"
    key="81"
  />
  <i
    className="rain"
    key="82"
  />
  <i
    className="rain"
    key="83"
  />
  <i
    className="rain"
    key="84"
  />
  <i
    className="rain"
    key="85"
  />
  <i
    className="rain"
    key="86"
  />
  <i
    className="rain"
    key="87"
  />
  <i
    className="rain"
    key="88"
  />
  <i
    className="rain"
    key="89"
  />
  <i
    className="rain"
    key="90"
  />
  <i
    className="rain"
    key="91"
  />
  <i
    className="rain"
    key="92"
  />
  <i
    className="rain"
    key="93"
  />
  <i
    className="rain"
    key="94"
  />
  <i
    className="rain"
    key="95"
  />
  <i
    className="rain"
    key="96"
  />
  <i
    className="rain"
    key="97"
  />
  <i
    className="rain"
    key="98"
  />
  <i
    className="rain"
    key="99"
  />
  <i
    className="rain"
    key="100"
  />
  <i
    className="rain"
    key="101"
  />
  <i
    className="rain"
    key="102"
  />
  <i
    className="rain"
    key="103"
  />
  <i
    className="rain"
    key="104"
  />
  <i
    className="rain"
    key="105"
  />
  <i
    className="rain"
    key="106"
  />
  <i
    className="rain"
    key="107"
  />
  <i
    className="rain"
    key="108"
  />
  <i
    className="rain"
    key="109"
  />
  <i
    className="rain"
    key="110"
  />
  <i
    className="rain"
    key="111"
  />
  <i
    className="rain"
    key="112"
  />
  <i
    className="rain"
    key="113"
  />
  <i
    className="rain"
    key="114"
  />
  <i
    className="rain"
    key="115"
  />
  <i
    className="rain"
    key="116"
  />
  <i
    className="rain"
    key="117"
  />
  <i
    className="rain"
    key="118"
  />
  <i
    className="rain"
    key="119"
  />
  <i
    className="rain"
    key="120"
  />
  <i
    className="rain"
    key="121"
  />
  <i
    className="rain"
    key="122"
  />
  <i
    className="rain"
    key="123"
  />
  <i
    className="rain"
    key="124"
  />
  <i
    className="rain"
    key="125"
  />
  <i
    className="rain"
    key="126"
  />
  <i
    className="rain"
    key="127"
  />
  <i
    className="rain"
    key="128"
  />
  <i
    className="rain"
    key="129"
  />
  <i
    className="rain"
    key="130"
  />
  <i
    className="rain"
    key="131"
  />
  <i
    className="rain"
    key="132"
  />
  <i
    className="rain"
    key="133"
  />
  <i
    className="rain"
    key="134"
  />
  <i
    className="rain"
    key="135"
  />
  <i
    className="rain"
    key="136"
  />
  <i
    className="rain"
    key="137"
  />
  <i
    className="rain"
    key="138"
  />
  <i
    className="rain"
    key="139"
  />
  <i
    className="rain"
    key="140"
  />
  <i
    className="rain"
    key="141"
  />
  <i
    className="rain"
    key="142"
  />
  <i
    className="rain"
    key="143"
  />
  <i
    className="rain"
    key="144"
  />
  <i
    className="rain"
    key="145"
  />
  <i
    className="rain"
    key="146"
  />
  <i
    className="rain"
    key="147"
  />
  <i
    className="rain"
    key="148"
  />
  <i
    className="rain"
    key="149"
  />
</div>
`;
